plugin/config.json

```diff
@@ -53,7 +53,11 @@
       "description": "Log level - info, warn, error or debug",
       "value": "info",
       "Settable": [ "value"]
-	}
-	]
+	},
+	{"name": "VDVS_DISCOVER_VOLUMES",
+      "description": "non-empty if pre-existing volumes need to be discovered ",
```

Comment:
It seems tabs and spaces are being mix-used which caused inconsistent alignment.